gordo_components/model/models.py

```diff
@@ -902,3 +776,80 @@ def score(
         out = self.transform(X)
 
         return explained_variance_score(out[:, : X.shape[1]], out[:, X.shape[1] :])
+
+
+def get_TimeseriesGenerator(data, batch_size, lookback_window, lookahead):
```

Comment:
I find `get_TimeseriesGenerator` a touch strange, in that this is basically a wrapper to instantiating `TimeseriesGenerator`. It doesn't follow standard naming convention for functions (snake case). I suggest either create a fully qualified wrapper `TimeseriesGenerator(sequence.TimeseriesGenerator)` and make this function's signature the `__init__` or classmethod of some kind, or if you prefer to have a free function for this, think the naming at least should be fixed.

Reply:
Also, if we want to continue to type our parameters in the signature.. not sure how strict we're being with that.

Reply:
We should type them, I update.

Reply:
I want it as a free function, and I will change the name.

Reply:
What about `get_keras_sequence` since the point is really that it returns a Keras Sequence object? Idk, naming is hard.

Reply:
I went with @flikka 's suggestion `create_keras_timeseriesgenerator`